src/resources/Alteration/C.ts

```diff
@@ -12,10 +12,10 @@ export const AlterationResourceC: ResourceIF = {
     entityType: CorpTypeCd.CONTINUE_IN,
     priority: false
   },
-  // Conditionally used in place of filingData when switching from BC (or C) to ULC:
+  // Conditionally used in place of filingData when switching from or C to CUL:
   additionalFilingData: {
     filingTypeCode: FilingCodes.ALTERATION_BC_TO_ULC,
-    entityType: CorpTypeCd.BC_COMPANY,
```

Comment:
Does this need correcting in the other resource files (CBEN, CCC, CUL)?

Reply:
I believe no - only C/BC can alter to CUL/ULC

Reply:
Yes, you are right: there is no `additionalFilingData` for CBEN, CCC, CUL. 👍 

Please fix typo in comment ("from or C to CUL").

Reply:
Thank you, fixed the typo